pyat/at/acceptance/boundary.py

```diff
@@ -259,14 +262,22 @@ def radial_boundary(mask, grid):
             bnd[:, i] = search_bnd(ma, sa)
         return bnd
 
+    some_survived = True
     if not numpy.any(mask):
```

Comment:
Implementing this check in search_bnd function could potentially make the code more compact what do you think?

Reply:
Dear @swhite2401,

no objection. My only issue is to make it proceed even if it finds a ZERO. I am open to any alternative that avoids the error. I do not know the code enough so I proposed something that could work, simply. 
Please push your proposed solution, or a modified code. I will test it!

thank you

Reply:
Ok I take over